eng/pipelines/runtime.yml

```
@@ -964,7 +964,7 @@ extends:
                     eq(variables['isRollingBuild'], true))
 
       #
-      # Android devices
+      # Android arm64 devices and x64 emulators
       # Build the whole product using CoreCLR and run libraries tests
```

```suggestion
      # Build the whole product using CoreCLR and run functional tests
```

I think we should be specific that this will be just for smoke/functional tests for now.